requirements.txt

```diff
@@ -1,2 +1,3 @@
 isort==5.10.1
 black==22.3.0
+pylint==2.17.2
```

Comment:
Is there any other reason to stick to this version than that we use it in avocado? Maybe since we are consolidating the checks around the all avocados projects, we could upgrade pylint to the latest version, which is 3.0.3. I did a quick check on avocado and only issue which we have there is [C1805](https://pylint.pycqa.org/en/latest/user_guide/messages/convention/use-implicit-booleaness-not-comparison-to-zero.html) which IMO won't be difficult to solve. What do you think?

Reply:
Well, it's really a matter of working with a more "fixed target" than the latest and greatest.  The usual approach I've followed is to start with the versions used in Avocado, and attempt make them common across all repos by using avocado-static-checks.

Then we should be able to easily update the versions and work on the extra checks, etc.